### PR TITLE
Closes Issue #53

### PR DESCRIPTION
**Pull request by zachlankton**
Closes Issue #53

Related Backend PR: https://github.com/mssiegel/frempco-server/pull/10

I've made these changes (including the backend PR) live on my temp server: [http://208.87.134.100:3000](http://208.87.134.100:3000)

### CODE REVIEW

File: src/components/pages/TeachersPage/ActivateButton.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default function ActivateButton({ socket, classroomName }) {
       'Type YES to confirm you want to deactivate the classroom.',
     );
     if (confirmation?.toUpperCase() === 'YES') {
-      socket.emit('deactivate classroom', { classroomName });
+      socket.emit('deactivate classroom');
```
Comment (zachlankton) [inline src/components/pages/TeachersPage/ActivateButton.tsx:21]: This calls `deleteClassroom` on the backend which was refactored in Backend PR: https://github.com/mssiegel/frempco-server/pull/9#discussion_r816717887

It is not needed to pass in the `classroomName` anymore.

**Comment by zachlankton** [comment]
Fair Point, I thought this was the intention as the current disconnect code in the backend deletes the classroom when the teacher disconnects.  So I was trying to follow suit.

I can remove the code that automatically kicks students when the teacher disconnects.

However, For teacher buy in, I think they will want to be able to get back in and resume viewing chats.  This is what I was think when I commented https://github.com/mssiegel/frempco-client/issues/53#issuecomment-1054153834 in the issue.  I used students as an example in my comment, but it is clear that teachers would absolutely benefit from some session/state persistence in this situation.

File: src/components/pages/TeachersPage/ActivateButton.tsx (hunk 1 above)
Comment (mssiegel) [reply src/components/pages/TeachersPage/ActivateButton.tsx:21]: So amazing you found this!!!!

**Review by mssiegel — APPROVED** [review]
This is fantastic!!! I left one [minor comment on the backend PR](https://github.com/mssiegel/frempco-server/pull/10#discussion_r818296149). Once that's addressed, feel free to merge this front end and backend Pr.

**Pull request merged**